### DISCUSSION TITLE
remap function

### DISCUSSION THREAD

**Discussion by Sulfurix74** [nesbox/TIC-80#2438] [Q&A] [answered]
can I animate more than one tile withe the same remap function?

**Comment by Skeptim** [comment, marked as answer]
Yes.
If I take the remap demo [here](https://github.com/nesbox/TIC-80/wiki/map#examples) as an example
```lua
	if tile==yourWaterfallTile then
		outTile=outTile+math.floor(gameTicks*speed)%frames
	end
```
animates `yourWaterfallTile` if you add an other `elseif` corresponding to an other tile you can animate this other tile.

**Reply by Sulfurix74** [reply]
thank u
